packages/firestore/src/api/timestamp.ts

```diff
@@ -92,4 +94,21 @@ export class Timestamp {
       ')'
     );
   }
+
+  // Overriding valueOf() allows Timestamp objects to be compared in JavaScript using the
+  // arithmetic comparison operators, such as < and >.
+  // https://github.com/firebase/firebase-js-sdk/issues/2632
+  //
+  // This method returns a string of the form <seconds>.<nanoseconds> where <seconds> is translated
```

Comment:
Can you move this comment into the method? You are describing implementation details. The consumer of the method should only now that the `valueOf()` return type should sort naturally.

Reply:
Done